### ISSUE TITLE
Fix this error message

### ISSUE THREAD

**Issue by afred** [WGBH/openvault#1129]
```
Openvault::Pbcore::AssetRelationshipBuilder::UnhandledRelationType: Openvault::Pbcore::AssetRelationshipBuilder::UnhandledRelationType
```
Not very useful.. i think the error object itself is getting output twice, instead of the error type followed by it's message.

**done when**
this type of error describes what type of relation is unhandled... i.e. if relation between a `Transcript` and a `Series` is unhandled, then include that info in the error message.

**Comment by mccalluc** [comment]
Is this still a problem? / Is there a reproducer? `AssetRelationshipBuilder::UnhandledRelationType` looks right...

**Comment by afred** [comment]
closed by c5064e0a15
